### PR TITLE
Update Kafka docs

### PR DESCRIPTION
**Pull request by samgnaniah**
Update Kafka docs.

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ The following diagram illustrates all the required functionality of the Kafka se
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
+## Set up Kafka
+
+Before you begin, set up Kafka by following the instructions in the [Setting up Kafka documentation](setting-up-kafka.md).
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md:19]: Shouldn't this be as follows:
"Before you begin, set up Kafka by following the instructions in [Setting up Kafka](setting-up-kafka.md)"

Or,
"Before you begin, follow the instructions to [set up Kafka](setting-up-kafka.md)"

Or,
"Before you begin, [set up Kafka](setting-up-kafka.md) for the Micro Integrator"

wdyt?

File: en/micro-integrator/docs/references/connectors/kafka-connector/kafka-inbound-endpoint-example.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ The following diagram illustrates all the required functionality of the Kafka se
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
+## Set up Kafka
+
+Before you begin, set up Kafka by following the instructions in the [Setting up Kafka documentation](setting-up-kafka.md).
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/references/connectors/kafka-connector/kafka-inbound-endpoint-example.md:17]: Same as above comment.

File: en/micro-integrator/docs/references/connectors/kafka-connector/setting-up-kafka.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 To use the Kafka connector, download and install [Apache Kafka](http://kafka.apache.org/downloads.html). Before you start configuring the Kafka you also need WSO2 EI and we refer to that location as <PRODUCT_HOME>.
 
-> **Note**: The recommended version is [kafka](https://www.apache.org/dyn/closer.cgi?path=/kafka/1.0.0/kafka_2.12-1.0.0.tgz). For all available versions of Kafka that you can download, see https://kafka.apache.org/downloads. The recommended Java version is 1.8.
+> **Note**: The recommended version is [Kafka 2.12-1.0.0](https://www.apache.org/dyn/closer.cgi?path=/kafka/1.0.0/kafka_2.12-1.0.0.tgz). For all available versions of Kafka that you can download, see https://kafka.apache.org/downloads. The recommended Java version is 1.8.
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/references/connectors/kafka-connector/setting-up-kafka.md:5]: Shouldn't this be removed?
"The recommended Java version is 1.8."
I assume, the first sentence provides the recommended Kafka version.

**Review by nilminiwso2 — APPROVED** [review]

File: en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md (hunk 1 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md]: Fixed.

File: en/micro-integrator/docs/references/connectors/kafka-connector/kafka-inbound-endpoint-example.md (hunk 2 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/references/connectors/kafka-connector/kafka-inbound-endpoint-example.md]: Fixed.

File: en/micro-integrator/docs/references/connectors/kafka-connector/setting-up-kafka.md (hunk 3 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/references/connectors/kafka-connector/setting-up-kafka.md:5]: I assumed we need Java as well for that version of Kafka to run.

Comment (nilminiwso2) [reply en/micro-integrator/docs/references/connectors/kafka-connector/setting-up-kafka.md:5]: oh.. right .. I missed that one.

**Pull request merged**